src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Contracts/Completion.cs

```diff
@@ -74,5 +95,15 @@ public class CompletionItem
         /// resolve request.
         /// </summary>
         public object Data { get; set; }
+
+        /// <summary>
+        /// Exposing a command field for a completion item for passing telemetry
+        /// </summary>
+        public Command Command { get; set; }
+
+        /// <summary>
+        /// Whether this completion item is preselected or not
+        /// </summary>
+        public bool? PreSelect { get; set; }
```

Comment:
Minor: is this casing what's sent? E.g. it's really `preSelect` from the vscode side?

Reply:
In VSCode, the property is "preselect", so I should rename it to "Preselect" as other property here.